addons/point_of_sale/tools/list_pos_modules.py

```diff
@@ -0,0 +1,12 @@
+import os
+import re
+
+def get_folders(path, regex):
```

Comment:
can you check if we can use `get_modules` instead? (`from odoo.modules.module import get_modules`)

Reply:
it looks like the import from odoo doesn't work in that directory